db/migrate/20190103093409_add_deployment_option_to_policy.foreman_openscap.rb

```diff
@@ -6,6 +6,7 @@ def up
         policy.update_attribute(:deploy_by, 'puppet')
       end
     end
+    ForemanOpenscap::Policy.reset_column_information
```

Comment:
I tried manually updating this and it failed. I also tried removing the later change_column and the migration ran but did not update the column for any records. After updating the values manually I was able to update the column in the database.

Reply:
Is there a reason you don't just do :default => "puppet"?

Reply:
Mainly because 'puppet' is not meant to be a default value, but rather one of the options. But I may do it eventually if I do not find another solution. Thanks for testing.

Reply:
@dforste did you have a chance to test the last version of the patch? does it help in your environment?

Reply:
Yes I had the patch work in my environment.